internal/impl/kube.go

```diff
@@ -489,6 +495,82 @@ func GenerateYAMLs(image string, app *protos.AppConfig, depId string, cfg *KubeC
 	return nil
 }
 
+// header returns the informational header at the top of a generated YAML file.
+func header(app *protos.AppConfig, cfg *KubeConfig, depId, filename string) (string, error) {
+	header := template.Must(template.New("header").Parse(`# This file was generated by "weaver kube" version {{.ToolVersion}} for the following
```

Comment:
I was thinking whether long term we should also provide more k8s stuff. E.g., for every deployment, stuff like cpu/memory request/limit, env variables set for the pods, min/max #replicas, volume mounts. Maybe for HPA, which rules apply. If someone takes a quick look at the header can get a good understanding of the high-level things that would happen if they deploy the yaml.

Reply:
Sounds like a good idea to me! I'll do this in a future PR.